fixed-price-subscriptions/server/ruby/server.rb

```diff
@@ -83,13 +85,15 @@
   # Set the default payment method on the customer
   Stripe::Customer.update(
     data['customerId'],
-    invoice_settings: { default_payment_method: data['paymentMethodId'] }
+    invoice_settings: {
+      default_payment_method: data['paymentMethodId']
+    }
   )
 
-  invoice =
-    Stripe::Invoice.retrieve(
-      { id: data['invoiceId'], expand: %w[payment_intent] }
-    )
+  invoice = Stripe::Invoice.retrieve({
+    id: data['invoiceId'],
```

Comment:
Maybe we should even pull all of the variable values out of the `data` up front? I'm not sure what would be cleaner/less noisy. 🤔

Reply:
Good idea, I'll try to do a bit more clean up there later too.